pom.xml

```diff
@@ -532,7 +532,7 @@
             <dependency>
                 <groupId>com.facebook.presto.hadoop</groupId>
                 <artifactId>hadoop-apache2</artifactId>
-                <version>2.7.4-5</version>
+                <version>2.7.4-7-SNAPSHOT</version>
```

Comment:
I noticed that the SNAPSHOT version has been uploaded to the maven repository, hence using this temporarily to run tests. Once the hadoop-apache2 release issue is resolved, I would update it to use release version.